serving/samples/helloworld-kotlin/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM openjdk:8-jdk-alpine
+
+CMD ["gradle"]
+
+ENV GRADLE_HOME /opt/gradle
```

Comment:
@pzmrzy is there a reason for not using `gradle` official image(s) ? (as it really does the same thing https://github.com/keeganwitt/docker-gradle/blob/bd6b0605196488734d8d61d6a00a4ca96846d2e7/jdk8/Dockerfile)

Reply:
@vdemeester I think no, I did't find this before, I just search on github for a gradle docker image

Reply:
@pzmrzy I guess we can do a follow-up to use an official image then :wink:

Reply:
@vdemeester Just update the image and test again

Reply:
@pzmrzy Can we use the official `gradle` image for this? Lines 1-34 can be replaced by:
```
FROM gradle
```